### PR TITLE
Updated and new label properties on admin1 and admin2

### PR DESCRIPTION
**Pull request by stepps00**
Fixes the Saudi Arabia portion of https://github.com/whosonfirst-data/whosonfirst-data/issues/1640.

- Updated existing `label` properties
- Adds localized Arabic longname and placetype labels, as well as Arabic (Latin) placetype

No PIP work required. Can merge once approved.

### CODE REVIEW

File: data/856/768/57/85676857.geojson [hunk 1]
```diff
@@ -5,26 +5,26 @@
...
-        "\u0645\u0643\u0629 \u0627\u0644\u0645\u0643\u0631\u0645\u0629"
-    ],
-    "label:ara_x_placetype_preferred":[
-        "\u0645\u0646\u0637\u0642\u0629 \u0625\u062f\u0627\u0631\u064a\u0629"
+    "label:ara_latn_x_preferred_placetype":[
+        "muhafazah"
     ],
-    "label:eng_x_placetype_longname":[
-        "Makkah Region"
+    "label:ara_x_preferred_longname":[
```
Comment (nvkelso) [inline data/856/768/57/85676857.geojson]: This seems very long, can you confirm it please?

Comment (stepps00) [reply data/856/768/57/85676857.geojson]: "مِـنْـطَـقَـة مَـكَّـة الـمُـكَـرَّمَـة"

Hmm.. looks like that translation is correct, but includes extra "\u0640" characters.. I've updated it in a new commit and will double-check all Arabic translations in this branch.

Comment (stepps00) [reply data/856/768/57/85676857.geojson]: looks like this is the only case - the other Arabic longnames look correct.

**Comment by stepps00** [comment]
This PR is good to go, filed a follow-up issue here: https://github.com/whosonfirst-data/whosonfirst-data/issues/1641

**Pull request merged**